source/expressions.tex

```diff
@@ -2823,8 +2823,8 @@
     typename-specifier \terminal{(} \opt{expression-list} \terminal{)}\br
     simple-type-specifier braced-init-list\br
     typename-specifier braced-init-list\br
-    postfix-expression \opt{\terminal{.} \terminal{template}} id-expression\br
-    postfix-expression \opt{\terminal{->} \terminal{template}} id-expression\br
+    postfix-expression \terminal{.} \opt{\terminal{template}} id-expression\br
+    postfix-expression \terminal{->} \opt{\terminal{template}} id-expression\br
```

Comment:
I omitted this one because it should make no difference to the rendered output.

Reply:
Right, that was precisely a reason in favour for me in order to keep diffs down, but I realize that it makes the Editor's Report more awkward, so I'll remove this and the next commit.